requirements.txt

```diff
@@ -8,3 +8,5 @@ pybind11
 pytest
 pytest-cov
 pytest-mock
+wheel
+matplotlib
```

Comment:
Why do we need wheel and matplotlib as package requirements?

Reply:
I needed `wheel` to run `bdist_wheel`. `matplotlib` can be removed, I just added it to make the scaling plots.

Reply:
Why did you need to build a wheel here? You should be able to avoid the wheel with the editable pip-installation build instructions from [here](https://github.com/vincentmr/pennylane-lightning-kokkos/tree/feature/kokkos/initarguments#installation):

For example, to build the OpenMP backend
```
BACKEND="OPENMP" python -m pip install -e .
```

Or for the threads backend

```
BACKEND="THREADS" python -m pip install -e .
```

Reply:
Got it. I didn't know about this option.